### PR TITLE
Add tracking for the screen kit

### PR DESCRIPTION
**Pull request by skarbat**
Add tracking when the screen kit is detected.
I doubt this API is the right way to query this information though...
Any suggestions @radujipa @Ealdwulf ?

### CODE REVIEW

**Comment by skarbat** [comment]
[(1) As a data scientist, I want to be able to identify which users have a Kano Computer screen](https://trello.com/c/y4amebR0/821-1-as-a-data-scientist-i-want-to-be-able-to-identify-which-users-have-a-kano-computer-screen)

File: bin/kano-ui-autostart [hunk 1]
```diff
@@ -38,6 +38,13 @@ function track_startup
     # Report a startup event to Kano Tracker
     kano-tracker-ctl +1 startup
 
+    # If the kit comes with the Kano Screen Kit, track it
+    # FIXME: The query below needs testing on real hardware
+    screen_kit=`python -c "from kano_settings.system import display; print display.get_model()"`
+    if [screen_kit == "" ]; then
```
Comment (radujipa) [inline bin/kano-ui-autostart]: Testing this condition on a flashed SK seems incorrect as it returns `HDMI1`. Judging from the `/boot/config.txt` we have two edid names which we go by when detecting SKs: `MST-HDMI1`, `MST-HDMI1`.

So the following command will give you the codes above.
```
python -c "from kano_settings.system import display; print display.get_edid_name(use_cached=False)"
``` 

However, I would make a function in `kano_settings.display` that would return whether an SK is plugged which you could then use above.

**Comment by skarbat** [comment]
Thank you @radujipa ! I'll take it from here :)

**Comment by skarbat** [comment]
Changed to use the new API. Tested on the PI with fake data, it is now tracking it on boot.

**Comment by radujipa** [comment]
That looks good :shipit:

**Comment by skarbat** [comment]
Great, merging for testing tomorrow.

**Pull request merged**